.buckconfig

```diff
@@ -1,7 +1,10 @@
 [buildfile]
     includes = //BUILD_DEFS
 [ndk]
-    ndk_version = r10c
+    ndk_version = r10e
+    cpu_abis = armv7, arm64, x86, x86_64
+    gcc_version = 4.9
+    app_platform = android-21
```

Comment:
Probably because `android-21` was the first platform that contained the `arm64` ndk ABI.